onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -985,6 +1118,11 @@ Status MIGraphXExecutionProvider::Compile(const std::vector<onnxruntime::Node*>&
     std::string onnx_string_buffer;
     model_proto.SerializeToString(&onnx_string_buffer);
 
+    std::string name = "ort_compile_" + std::to_string(fused_node_index++) + ".onnx";
```

Comment:
Could you please change std::ofstream to STD C APIs(fopen/fwrite/fclose) ? ofstream doesn't work well in our environment.